### PR TITLE
Guide: Deploy Zola with Layer0

### PR DESCRIPTION
**Pull request by rishi-raj-jain**
Sanity check:

* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/getzola/zola/pulls) for the same update/change?

### CODE REVIEW

**Review by Keats — APPROVED** [review]
Never heard of that service but thanks!

File: docs/content/documentation/deployment/layer0.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
++++
+title = "Layer0"
+weight = 50
++++
+
+[Layer0](https://www.layer0.co) is an all-in-one platform to develop, deploy, preview, experiment on, monitor, and run your headless frontend. It is focused on large, dynamic websites and best-in-class performance through EdgeJS (a JavaScript-based Content Delivery Network), predictive prefetching, and performance monitoring. Layer0 offers a free tier.
```
Comment (0xpr03) [inline docs/content/documentation/deployment/layer0.md:6]: that's like an ad

Comment (Keats) [reply docs/content/documentation/deployment/layer0.md:6]: I know, but the other pages have the same kind of text so I didn't want to ask for changes to the PR. I'll remove all of them at once this week.

Comment (0xpr03) [reply docs/content/documentation/deployment/layer0.md:6]: ah well then ignore me

Comment (rishi-raj-jain) [reply docs/content/documentation/deployment/layer0.md:6]: @0xpr03 Thanks for the feedback! I've received a similar one as to what I contribute, and I'll definitely keep this in mind for any of my future contributions.
@Keats I'm happy to make those PR changes to serve the docs best.

**Pull request merged**